addons/l10n_jo/data/account.group.csv

```diff
@@ -0,0 +1,31 @@
+id,code_prefix_start,code_prefix_end,name,parent_id/id
+jo_group_1,100101,100199,100100 Liquidity,
```

Comment:
@Nadaseder

Reply:
The liquidity group should cover all bank and cash accounts, that's why I extended the end prefix, can you clarify your suggestion?

Thanks, 

@jco-odoo

Reply:
My suggestion was just to have technically the same thing with shorter codes.  (suppose people would define subgroups e.g.)

Reply:
but in that case if someone wants to use the standard set up it will not be appropriate, I suggest we have it fully configured with the account groups in this way, if someone wants to go the extra mile and change the account groups, he would do the necessary edits on the localized accounts groups  

@jco-odoo

Reply:
I don't see how my suggestion would be any different functionally.  It is just shorter. There is even  no need to put the code_prefix_end.